src/Chapter2.hs

```diff
@@ -741,7 +763,11 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]]
+contains _ [] = []
+contains e (x:xs)
+  | elem e x = x : contains e xs
+  | otherwise = contains e xs
```

Comment:
Nice solution! 💯 
You can also utilise the `filter` function, which actually does exactly what you coded here 🧠